src/sentry/notifications/notifications/rules.py

```diff
@@ -157,3 +157,80 @@ def get_log_params(self, recipient: Team | User) -> Mapping[str, Any]:
             "target_identifier": self.target_identifier,
             **super().get_log_params(recipient),
         }
+
+
+class ActiveReleaseAlertNotification(AlertRuleNotification):
+    message_builder = "ActiveReleaseIssueNotificationMessageBuilder"
+    metrics_key = "release_issue_alert"
+    notification_setting_type = NotificationSettingTypes.ISSUE_ALERTS
+    template_path = "sentry/emails/active-release"
```

Comment:
Template path where we pull in the email template file.

Reply:
Might make sense to either point this to `sentry/emails/error` for now, or copy `sentry/emails/error` into this file until we have the new design